modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/GlobalJSCETsConfiguration.tsx

```diff
@@ -377,6 +377,57 @@ function ExtensionRow({
 		},
 	];
 
+	const getScriptElementAttributesScope = () => {
```

Comment:
Hey @thiago-buarqque wouldn't these frontend changes need to be behind a feature flag? Is there a check somewhere else that is handling this?

Reply:
@ethib137 Oops, you are right. I should add the feature flag check [here](https://github.com/liferay-platform-experience/liferay-portal/pull/553/files#diff-cd84f38002311fafb756b32b7bdf1626b528626ffa0ed560d859ccba410780c8R54) and [here](https://github.com/liferay-platform-experience/liferay-portal/pull/553/files#diff-0e0293522d5c45f8ab182a328d4eb5aaeb917b406c3a965d31db9cb1ac91ba9aR446). I think checking on the backend is better than on the frontend in terms of performance.

Reply:
Done!

See [this](https://github.com/liferay-platform-experience/liferay-portal/pull/553/commits/310be97ac4972bd31896b745524eb152d3c3a8ff#diff-0e0293522d5c45f8ab182a328d4eb5aaeb917b406c3a965d31db9cb1ac91ba9aR450) and [this](https://github.com/liferay-platform-experience/liferay-portal/pull/553/commits/310be97ac4972bd31896b745524eb152d3c3a8ff#diff-cd84f38002311fafb756b32b7bdf1626b528626ffa0ed560d859ccba410780c8R58)